docs/framework/deployment/windows/10.md

```diff
@@ -0,0 +1,17 @@
+# Installing the .NET Framework on Windows 10
+
+You may need the .NET Framework to run an application on Windows 10. You can use the following instructions to help you. The instructions depend on which version of the .NET Framework you need.
```

Comment:
I'd start this by saying the .NET Framework 4.6.2 is included with Windows 10 Anniversary Update, and that you only need to run a .NET Framework installation to install a later version.